backend/utils/pagination.py

```
@@ -10,39 +8,34 @@
-                    WHERE multiplier_at_creation IS NOT NULL
-                """)
-                
-                cursor.execute("""
-                    UPDATE contributions_contribution
-                    SET frozen_global_points = points
-                """)
-            
-            # Try again with the original queryset after fixing the database
-            return super().paginate_queryset(queryset, request, view)
\ No newline at end of file
+            logger.error(
```

Ruff (0.15.15)

[warning] Logging `.exception(...)` should be used instead of `.error(..., exc_info=True)`

(G201)